packages/core/components/HeaderRibbon/tutorials/GenerateManifest.ts

```diff
@@ -0,0 +1,13 @@
+import Tutorial from "../../../entity/Tutorial";
+
+export const GENERATE_MANIFEST_TUTORIAL = new Tutorial("Generating manifests")
+    .addStep({
+        targetId: Tutorial.FILE_LIST_ID,
+        message:
+            "Select a file by left-clicking or multiple by holding Shift or Ctrl and clicking multiple files (any will do for this tutorial)",
```

Comment:
I'm not very familiar with this feature, but I would explain what it is in the first sentence.

> You can generate CSV manifests containing a list of FMS files. Start by selecting any number of files from the file list.